src/components/button/README.md

```diff
@@ -0,0 +1,49 @@
+# ui-component-button
+Our main button component.
+
+## Installation
+To install this module:
+
+`npm install [git repo]`
```

Comment:
Planning to address this when we make the components **cherrypickable** < new word